web/src/user_profile.js

```diff
@@ -9,6 +9,7 @@ import * as browser_history from "./browser_history";
 import * as buddy_data from "./buddy_data";
 import * as channel from "./channel";
 import * as components from "./components";
+import {DropdownListWidget} from "./dropdown_list_widget";
```

Comment:
I am on path to remove `DropdownListWidget` from code. Once #26067 is merged. You can implement this using `tippy`.

Reply:
I do plan to make a lot of abstraction in this code path. So, I am not it is the best idea for you to work on this right now.

Reply:
Okay, then I will just make the other changes you suggested and will not change this to tippy for now

Reply:
@amanagr since the #26067 has been merged, should I continue working on this or you still want to make some more changes ?

Reply:
There will be changes but I think you can implement this using tippy.

Reply:
We've now removed the old widget, so you can migrate this PR.